extensions/Cake.Board.AzureBoards.yml

```diff
@@ -3,7 +3,7 @@ Name: Cake.Board.AzureBoards
 NuGet: Cake.Board.AzureBoards
 Assemblies:
 - "/**/Cake.Board.AzureBoards.dll"
-Repository: https://github.com/cake-contrib/Cake.Board/
+Repository: https://github.com/nicolabiancolini/Cake.Board/
```

Comment:
Why the change here?

Reply:
I don't know for sure. I'm guessing that's what was in the nuspec in the older package. Give me a few minutes to research...

Reply:
Ok I figured it out. It's a little complicated:
1. the nuspec of the most recent **prerelease** package contains URL which points to the repo under the cake-contrib organization. This explains the `https://github.com/cake-contrib/Cake.Board/` value which was previously in the YAML
2. the nuspec in the older "stable" package contains the URL which points to the original author's repo (presumably because at that time the repo had not been transferred to the cake-contrib organization yet)
3. AddinDiscoverer has logic to check if an addin has been transferred to the cake-contrib organization and if so, automatically update the URL. In this case however, this logic fails because the name of the repo does not match the name of the addin: the repo is "Cake.Board" and the addin is "Cake.Board.AzureBoards".

Reply:
So in conclusion, the different values are explained because the nuspec changed between July 2019 and September 2019 AND ALSO AddinDisco was unable to automatically fix the URL because of the discrepancy between the repo name and the addin name.

Make sense?

Reply:
Yes, makes sense